main_window/main_window.py

```diff
@@ -71,7 +86,7 @@ def __init__(self, parent=None):
 
         # Show P&ID Diagram handler
         self.pid_window = PIDWindow()
-        self.ui.showPIDButton.clicked.connect(self.open_pid_window)
+        #self.ui.showPIDButton.clicked.connect(self.open_pid_window)
```

Comment:
Could you add this button back to the UI? It should have the correct functionality.